spec/integration/rights_spec.rb

```diff
@@ -1,29 +1,51 @@
 # frozen_string_literal: true
 
+require "ffi-icu"
 require "rack/test"
 require "shared_examples"
 
 RSpec.describe "/rights" do
   include Rack::Test::Methods
+  # Use ICU collator to try to approximate Sequel's collation of underscore vs Ruby's
+  let(:collator) { ICU::Collation::Collator.new("en") }
 
-  context "with a valid HTID" do
-    before(:each) { get(rights_api_endpoint + "rights/test.pd_google") }
-    it_behaves_like "nonempty rights response"
-  end
+  describe "/rights_log/HTID" do
+    context "with a valid HTID" do
+      before(:each) { get(rights_api_endpoint + "rights/test.pd_google") }
+      it_behaves_like "nonempty rights response"
+    end
 
-  context "with an invalid HTID" do
-    before(:each) { get(rights_api_endpoint + "rights/bogus.no_such_id") }
-    it_behaves_like "empty response"
+    context "with an invalid HTID" do
+      before(:each) { get(rights_api_endpoint + "rights/bogus.no_such_id") }
+      it_behaves_like "empty response"
+    end
   end
 
   context "with no HTID" do
     before(:each) { get(rights_api_endpoint + "rights") }
     it_behaves_like "nonempty rights response"
 
-    it "obeys default sort order `time ASC`" do
+    it "obeys default sort order `namespace, id, time`" do
```

Comment:
referring to the comments below - I'd be inclined to keep this and drop the tests for the return type from `default_order`